ui/lib/apps/Statement/components/StatementSettingForm.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { useEffect, useState } from 'react'
+import {
+  Form,
+  InputNumber,
+  Skeleton,
+  Switch,
+  Input,
+  Slider,
+  Space,
+  Button,
+  Modal,
+} from 'antd'
+import { StatementConfig } from '@lib/client'
+import { ExclamationCircleOutlined } from '@ant-design/icons'
+import { useTranslation } from 'react-i18next'
+
+interface Props {
+  instanceId: string
+
+  onClose: () => void
+  onFetchConfig: (instanceId: string) => Promise<StatementConfig | undefined>
+  onUpdateConfig: (instanceId: string, config: StatementConfig) => Promise<any>
+  onConfigUpdated: () => any
+}
+
+type InternalStatementConfig = StatementConfig & {
+  keep_duration: number
+  max_refresh_interval: number
+  max_keep_duration: number
+}
+
+const REFRESH_INTERVAL_MARKS = {
```

Comment:
`REFRESH_INTERVAL_MINUTES = [1, 5, ...]`

Reply:
marks prop for Slider  should be an object, not accept an array, so I add an util method to convert the arr to object:

```js
const convertArrToObj = (arr: number[]) =>
  arr.reduce((acc, cur) => {
    acc[cur] = cur
    return acc
  }, {})

const REFRESH_INTERVAL_MARKS = convertArrToObj([1, 5, 15, 30, 60])
const KEEP_DURATION_MARKS = convertArrToObj([1, 2, 5, 10, 20, 30]
```